libs/hmpb/src/ballot_templates/nh_ballot_template.tsx

```diff
@@ -70,8 +70,8 @@ function Header({
 
   electionTitleOverride,
   electionSealOverride,
-  clerkSignatureImage,
-  clerkSignatureCaption,
+  clerkSignatureImage: clerkSignatureImageOverride,
```

Comment:
I added these to communicate that these are overrides now - could I get input on if you'd suggest more of a comment or even removing these variable renames? I can see an argument from all the above.

I decided to not rename the fields within the `NhPrecinctSplitOptions` type that provides these props to avoid making a backwards incompatible change that would require a migration. But open to that if we'd prefer